go/engine/login_provision.go

```diff
@@ -917,6 +919,13 @@ func (e *loginProvision) ensurePaperKey(ctx *Context) error {
 		}
 	}
 
+	// Load me so that keys will be up to date.
+	var err error
+	e.arg.User, err = libkb.LoadUser(libkb.LoadUserArg{Self: true, UID: e.arg.User.GetUID(), PublicKeyOptional: true, Contextified: libkb.NewContextified(e.G())})
```

Comment:
In the same vein as https://github.com/keybase/client/pull/6713, how about leaving this in because doing it another way would be more complicated?